### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by praizerema**
Review index.html
Review .gitignore
Review style.css
Review README.md

### CODE REVIEW

**Review by praizerema — COMMENTED** [review]
Good job. Approved.
Go ahead and merge your code to the main branch.

**OPTIONAL**
The title of your project should fit the description.
You should have an image in your README.md file showing the web page

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] The main idea is to use the "Changes required" title on your review and then do the review. Please, find at least 2 things to improve on your project.  🐶 
- [ ] Please, add at least two inline comments in a polite way and be nice, don't forget to add why is important the change you request. 🐱 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header id="header">
+      <nav id="nav-bar">
+        <img
+          id="header-img"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          width="200"
+          alt=""
```
Comment (praizerema) [inline index.html:16]: - [ ] Please provide an alternative text for the image element example `alt="my image"`.They are used to describe the image. One of the main purposes of `alt` is for the benefit of visually impaired users who use screen readers when browsing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <h2>Request For Newsletters</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            type="email"
+            id="email"
+            placeholder="Email"
+            name="email"
+            required
+          />
+          <input type="submit" id="submit" value="submit" />
```
Comment (praizerema) [inline index.html:66]: - [ ] Please change the input type `type ="submit"` here is invalid. You can use `type ="button"`.

**Review by praizerema — COMMENTED** [review]
### Changes Required

Hi @praizerema, 
Good job so far. But there are a few changes you need to work on.

- [ ] Please make changes to the title of your project. HTML title is important because it is used to display the webpage title in the browser and also helps in Search Engine Optimization
- [ ] Please provide an alternative text for the image element example alt="my image".They are used to describe the image. One of the main purposes of alt is for the benefit of visually impaired users who use screen readers when browsing.
- [ ] Please change the input type type ="submit" here is invalid. You can use type ="button".

### Optional Suggestions
- [ ] Kindly attach an image in your README.md file showing the web page. This makes it easier for anybody on the repository to know what to expect from the code.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**